module1/homework/removeVowels/vowel.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+const std::string vowels{"aeiouy"};
+
+void filterString(std::string &, const std::string &);
+
+void removeVowels(std::vector<std::string> &);
```

Comment:
Is good to add empty line at the end of the file.

Reply:
Depends, in gerrite system that I use at work empyt line at the end are marked as wrong